src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Repository/LocationMetaRepository.cs

```diff
@@ -113,16 +114,24 @@ FROM read_csv('{dataSetVersionPathResolver.CsvDataPath(dataSetVersion):raw}', AL
                         .Where(o => !existingRowKeys.Contains(o.GetRowKey()))
                         .ToList();
 
-                    var startIndex =
-                        await publicDataDbContext.LocationOptionMetas.CountAsync(token: cancellationToken) + 1;
+                    var startIndex = (await publicDataDbContext.Database
+                            .SqlQuery<int>($"SELECT nextval('public.\"LocationOptionMetas_Id_seq\"')")
```

Comment:
Could abstract a constant on `PublicDataDbContext` for `LocationOptionMetas_Id_seq` to re-use when interacting with the sequence in this method?

Similar thing can be done in the seeder